src/Application.php

```diff
@@ -162,7 +162,9 @@ public function refineUriSelection($cwd)
         $uri = $selfAliasRecord->uri();
 
         if (empty($uri)) {
-            $uri = $this->bootstrapManager()->selectUri($cwd);
+            if (!$uri = $this->config->get('options.uri')) {
```

Comment:
Style nit: factor this out into `$this->selectUri($cwd)`. Avoid putting assignments inside of `if` statements. Use `return $uri` to exit the selectUri() method early if you get a valid from config, and then otherwise `return $this->bootstrapManager()->selectUri($cwd);`